### PR TITLE
Seedlet: Update headstart annotations

### PR DESCRIPTION
**Pull request by jffng**
This PR partially addresses #2125 to include headstart annotations. It adds a home page layout and one blog post to the user's site, as well as setting the primary and social navigation. 

To test:
- Check out this PR onto your sandbox
- From your sandbox, run: `wp headstart run --user=[dotcom username] --url=[your Headstart test site] --theme=pub/seedlet`
- Visit the site and verify that the headstart content has populated your test site

### CODE REVIEW

File: seedlet/inc/headstart/en.json [hunk 1]
```diff
@@ -74,7 +60,7 @@
             "hs_custom_meta": "_hs_front_page",
             "hs_sharing": 1,
             "hs_like_status": 1,
-            "attachment_url": "https:\/\/headstartdata.files.wordpress.com\/2016\/06\/skyline-buildings-new-york-skyscrapers.jpg",
+            "attachment_url": "https://seedletdemo.files.wordpress.com/2020/06/ec867-plants_2x2.png",
```
Comment (kjellr) [inline seedlet/inc/headstart/en.json:63]:
```suggestion
            "attachment_url": "https://headstartdata.files.wordpress.com/2020/07/envelope.jpg",
```

I've uploaded a version of this to the headstartdata site. 👍

File: seedlet/inc/headstart/en.json [hunk 2]
```diff
@@ -60,12 +46,12 @@
             "hs_custom_meta": "_hs_first_post",
             "hs_sharing": 1,
             "hs_like_status": 1,
-            "attachment_url": "https:\/\/headstartdata.files.wordpress.com\/2016\/06\/pexels-photo-30732.jpg",
+            "attachment_url": "https://seedletdemo.files.wordpress.com/2020/07/envelope.png",
```
Comment (kjellr) [inline seedlet/inc/headstart/en.json:49]:
```suggestion
            "attachment_url": "https://headstartdata.files.wordpress.com/2020/07/envelope.jpg",
```

I've uploaded a higher-res version of this to the headstartdata site.

**Comment by kjellr** [comment]
Cool — this is looking pretty good. I generated higher-resolution versions of a couple of these images, but I think I need to do more (I noticed they're low-res on the theme demo too 😕). I'll try to find some time to get those ready later today, so let's hold onto this PR for now.

**Comment by ianstewart** [comment]
The homepage content "headstarted" as expected for me.

image

Looks like only [a social menu](https://github.com/Automattic/themes/blob/9b7a7ede6ca0a772971dfa644193d51b59bfd87e/seedlet/inc/headstart/en.json#L81) was added. I don't think it's a concern or blocker for how we currently use Headstart today though.

File: seedlet/inc/headstart/en.json (hunk 1 above)
Comment (kjellr) [reply seedlet/inc/headstart/en.json:63]: Actually, I'm not sure it even makes a difference where the images come from. 🤷‍♂️

I've updated the theme demo, and will update this PR in just a moment.

**Comment by kjellr** [comment]
Just pushed an update where I updated the demo content with fresh data from the demo site (including higher res, cleaned up images). Please note that I also removed the bottommost promo on the homepage from the demo, since I think the page flows a little more nicely without it.  

@jffng if you give this one more test, it should be ready to go.

**Comment by jffng** [comment]
Nice, thanks @kjellr and @ianstewart 🚀

**Pull request merged**